sale_order_type_whole_delivered_invoiceability/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module extends the functionality of the sale_order_whole_delivered_invoiceability 
+module by adding the option to prevent the invoicing of sales orders until all material 
+of the sales order type has been delivered.
```

Comment:
```suggestion
module by adding the option to prevent the invoicing of sales orders until all material 
has been delivered on the sales order types, being propagated from them where they
are used in the sales order.
```